cosmoz-omnitable-treenode-column.html

```diff
@@ -112,6 +121,52 @@
 			_tooltip: {
 				type: String,
 				computed: '_computeTooltip(filter)'
+			},
+
+			/**
+			 * `<paper-autocomplete>` `queryFn`
+			 * Search for indexOf query in full text of sources.
+			 * Highlights the matches.
+			 */
+			queryFn: {
+				type: Function,
+				value: function () {
+					return function (datasource, query) {
+						return datasource.map(item => {
+							let text,
+								value;
+
+							if (item == null) {
```

Comment:
This check can be done first, before let/value are declared (or even a .filter())